packages/backend-core/src/db/conversions.js

```diff
@@ -50,3 +50,8 @@ exports.getProdAppID = appId => {
   const rest = split.join(APP_DEV_PREFIX)
   return `${APP_PREFIX}${rest}`
 }
+
+exports.extractAppId = id => {
```

Comment:
what does this extract the app ID from? We would usually call the app ID the full `app_<uuid>` - might be worth naming this if it is for extracting the UUID from an app ID.